app/assets/javascripts/dul-argon-skin.js

```diff
@@ -318,4 +318,32 @@ Blacklight.onLoad(function() {
 
     });
 
+});
+
+
+Blacklight.onLoad(function() {
```

Comment:
I'd add this whole block to the existing Blacklight.onLoad() block above, and remove the commented-out $(window).load() function.